pkg/model/employees.go

```diff
@@ -41,17 +41,20 @@ type Employee struct {
 	DateOfBirth             *time.Time
 
 	// working info
-	WorkingStatus WorkingStatus
-	JoinedDate    *time.Time
-	LeftDate      *time.Time
-	AccountStatus AccountStatus
+	WorkingStatus     WorkingStatus
+	JoinedDate        *time.Time
+	LeftDate          *time.Time
+	AccountStatus     AccountStatus
+	EmployeePositions []EmployeePosition
+	ProjectMembers    []ProjectMember
```

Comment:
Preload field should be separated to other group

Reply:
okay